.github/workflows/tests.yml

```diff
@@ -27,8 +33,16 @@ jobs:
 
       - uses: ./go.mk/.github/actions/pre-check
 
-      - name: Run tests
+      - name: Run tests with default arguments
+        if: github.event.inputs.go_test_extra_args == ''
```

Comment:
Nice idea! But do we really need this `if`? Isn't it enough to leave GO_TEST_EXTRA_ARGS empty and it will run all the tests?

Reply:
I think empty string  would not override Makefile default as empty env var is usually handled same as unset. It's worth testing to be certain.